test/Api.Test/Models/Request/Accounts/KdfRequestModelTests.cs

```diff
@@ -1,65 +0,0 @@
-﻿using System.ComponentModel.DataAnnotations;
-using Bit.Api.Auth.Models.Request.Accounts;
-using Bit.Core.Enums;
-using Xunit;
-
-namespace Bit.Api.Test.Models.Request.Accounts;
-
-public class KdfRequestModelTests
-{
-    [Theory]
-    [InlineData(KdfType.PBKDF2_SHA256, 1_000_000, null, null)] // Somewhere in the middle
-    [InlineData(KdfType.PBKDF2_SHA256, 600_000, null, null)] // Right on the lower boundary
```

Comment:
💭 It would be nice to keep coverage for these lower and upper bound checks. What about converting these over to a `KdfSettingsValidatorTests` to test the static class `KdfSettingsValidator`?